server/build.gradle

```diff
@@ -68,6 +68,9 @@ dependencies {
     implementation('io.micrometer:micrometer-core:1.14.2')
     implementation('io.micrometer:micrometer-tracing:1.4.1')
     implementation('io.micrometer:micrometer-tracing-bridge-otel:1.4.1')
+
+    implementation('org.projectlombok:lombok:1.18.36')
```

Comment:
Added for annotation in `DefoldSdkServiceConfiguration` to avoid boilerplate code.